src/internet_identity/internet_identity.did

```diff
@@ -74,6 +74,7 @@ type DeviceData = record {
     key_type: KeyType;
     protection: DeviceProtection;
     origin: opt text;
+    meta_data: opt MetaDataMap;
```

Comment:
I've actually never seen "metadata" spelled in two words. Would a single-word "metadata" work for you?

Reply:
Yes, I'll change it.